src/vs/workbench/parts/debug/electron-browser/debugEditorContribution.ts

```diff
@@ -192,6 +200,42 @@ export class DebugEditorContribution implements IDebugEditorContribution {
 		});
 	}
 
+	private onFocusStackFrame(stackFrame: IStackFrame): void {
+		// Since step over, step out is a fast continue + break. Continue clears stack.
+		// This means we'll get a null stackFrame followed quickly by a valid stackFrame.
+		// Removing all decorators and adding them again causes a noticeable UI flash due to relayout and paint.
+		// We want to only remove inline decorations if a null stackFrame isn't followed by a valid stackFrame in a short interval.
+		clearTimeout(this.removeDecorationsTimeoutId);
```

Comment:
Explained the issue about deltaDecorators on other thread.  We need the timeout anyway since setDecorators gives us a nice diff for free. We just need to debounce the null between the continues, otherwise we end up removing all decorators and creating them again rather than only changing the subset that actually are different.